api/src/test/java/org/openmrs/module/fhir2/providers/BaseFhirProvenanceResourceTest.java

```diff
@@ -9,34 +9,17 @@
  */
 package org.openmrs.module.fhir2.providers;
 
-import static org.mockito.Mockito.when;
-
 import java.util.Arrays;
 import java.util.Date;
 
 import org.hl7.fhir.r4.model.CodeableConcept;
 import org.hl7.fhir.r4.model.Coding;
 import org.hl7.fhir.r4.model.DomainResource;
-import org.hl7.fhir.r4.model.Practitioner;
 import org.hl7.fhir.r4.model.Provenance;
-import org.hl7.fhir.r4.model.Reference;
-import org.mockito.Mock;
-import org.openmrs.User;
 import org.openmrs.module.fhir2.FhirConstants;
-import org.openmrs.module.fhir2.api.translators.PractitionerReferenceTranslator;
 
 public abstract class BaseFhirProvenanceResourceTest<T extends DomainResource> {
 	
```

Comment:
I don't think any of this was causing a problem in the end (not entirely sure, as I was just picking stuff off one by one), but none of it is needed / used, and removing it did not lead to any issues, so I removed it.